packages/core/src/contexts/refine/index.tsx

```diff
@@ -15,6 +15,7 @@ export const defaultRefineOptions: IRefineContextOptions = {
     undoableTimeout: 5000,
     warnWhenUnsavedChanges: false,
     liveMode: "off",
+    indexRoute: "",
```

Comment:
```suggestion
    indexRoute: "/",
```

I think, we can manage the refine default values in here.